vscode-extension/src/utils.ts

```diff
@@ -34,43 +36,28 @@ function getBinaryPathRelativeToPackageJson() {
     binaryPathRelativeToPackageJson = null;
   }
 
-  if (binaryPathRelativeToPackageJson) {
-    return path.join(
-      '.',
-      'node_modules',
-      'relay-compiler',
-      binaryPathRelativeToPackageJson,
-    );
-  }
-
-  return null;
+  return binaryPathRelativeToPackageJson;
```

Comment:
If we don't need to do any processing of `binaryPathRelativeToPackageJson` after deciding the path, we could probably just avoid the variable all together and just early return in each condition.